### DISCUSSION TITLE
Dudas, entregas CSS

### DISCUSSION THREAD

**Discussion by OscarIslasH** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#355] [Q&A] [answered]
¿Hasta cuando podemos hacer la entrega de la pactica de CSS?

Voy a empezar mañana por el trabajo y me gustaria saber la fecha limite para poder entregarla

**Comment by OmarVenturaP** [comment]
En el live coding de hoy mencionó el MC que mañana puede que suba el link para entrega según el domingo, pero es probable que de margen de entregarlo otro tiempo más, quizás una semana, así como la de HTML.

**Reply by OscarIslasH** [reply]
Con lo de la pagina de HTML se me dificulto bastante el tema del CSS, esperemos y si deje un poco mas de tiempo, apenas voy a ver los videos guardados de los lives

**Comment by doguedogue** [comment, marked as answer]
Mañana suben el link para cargar actividad css.

Entrega límite hasta domingo 13 de marzo.

Nota: Con probable aplazamiento de unos días mas, según lo que diga Dobel 🙈

**Reply by OscarIslasH** [reply]
A mi parecer es mas seguro que si, mas que nada por que apenas estaran terminando lo del html y lo del css es mas complicado.

Gracias por aliviarme un poco aunque no sea seguro jajaja.
